src/rtctools/data/csv.py

```diff
@@ -51,9 +52,21 @@ def load(fname, delimiter=",", with_time=False):
 
     :returns: A named numpy array with the contents of the file.
     """
+
+    def string2datetime(string: Union[str, bytes]):
```

Comment:
These two functions should be moved above to sit underneath the function `_boolean_to_nan`. Also, please use a leading underscore to make the functions private to this module. Finally, for consistence `_boolean_to_nan`, please use the word "to" rather than the number "2" in the function names.

Reply:
I'll move and rename the functions.